tests/common/random.rs

```diff
@@ -0,0 +1,331 @@
+//  * This file is part of the uutils coreutils package.
+//  *
+//  * For the full copyright and license information, please view the LICENSE
+//  * file that was distributed with this source code.
+
+#![allow(dead_code)]
```

Comment:
Do we still need this? Can we remove (or export) `random_char`?

Reply:
I removed it.

Reply:
I removed it.